release-notes/aws/10.0.0-rc1

```diff
@@ -0,0 +1,29 @@
+## :zap:  Giant Swarm Release 10.0.0-rc1 for AWS is now active for you! :zap:
+
+We are pleased to announce that Node Pools is now released! 
+
+Please see here the supported material which we recomend reading before getting started: https://docs.giantswarm.io/basics/nodepools
+
+We also kindly remind you that from this release and for a period of SIX months onwards, to give you a chance to move your clusters, we will be supporting both the 'Legacy' and the 'New' branches separately which will include:
+
+:older_adult: 'Legacy' Branch: 
```

Comment:
I'd highlight the new things first. So first the new release and then the legacy branch. Actually, do we need to talk about the legacy branch at all? For me it is enough to mention it in the sentence above. But this message here should probably focus on the node pools release only then.

Reply:
please see suggested changes